src/quantum/azext_quantum/tests/latest/utils.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+TEST_SUBS = "677fc922-91d0-4bf6-9b06-4274d319a0fa"
+TEST_RG = 'aqua-provider-validator'
```

Comment:
Do we want the code name "aqua" to be visible in distributed code? That could cause some confusion with other names, possibly.